modules/onnx/1.17.0/presubmit.yml

```
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+    - windows
+  bazel: [6.x, 7.x]
```

```suggestion
  bazel: [7.x, 8.x, rolling]
```
Please update bazel version